account_move_helper/models/res_partner.py

```diff
@@ -70,7 +72,8 @@ def _compute_debit_credit(self):
                 company_id = self._context.get('company_id', False)
                 if company_id:
                     domain.append(('company_id', '=', company_id))
-
+                if move:
+                    domain.append(('date', '<=', move.date))
                 rec.update({
                     field: sum(rec.env['account.move.line'].search(
```

Comment:
> rec.env['account.move.line']

should be **self.//env['account.move.line'] and defined as variable outside the for loop